Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/information/InformationPrivacyFragment.kt

```diff
@@ -27,7 +28,14 @@ class InformationPrivacyFragment : Fragment(R.layout.fragment_information_privac
 
     private fun setButtonOnClickListener() {
         binding.informationPrivacyHeader.headerButtonBack.buttonIcon.setOnClickListener {
-            (activity as MainActivity).goBack()
+
+            if (activity is (MainActivity)) {
```

Comment:
Can't we just `popBackstack()` here? It's not so nice that the fragment needs to know about the different activities.